### DISCUSSION TITLE
Possible v108 exploit

### DISCUSSION THREAD

**Discussion by NightScripts-API** [3kh0/ext-remover#394] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So recently I was trying the method by **Bypassi**  which looks something like this:

**javascript:let shim = false;var ids = prompt("haldlgldplgnggkjaafhelgiaglafanh  -- GOGUARDIAN ID").split(",");setInterval(()=>{ids.forEach((id)=> opener.chrome.developerPrivate.updateExtensionConfiguration({extensionId: id, fileAccess: shim}));shim = !shim;}, 1500);**

For this method it allows the _Extension_ to Turn **OFF** and **ON** the **GoGuardian** _extension_.

I found this type of glitch it seems where when you enter a **Locked** _Google_ _Forms_ and select the _Window_ _key_ on the **chromebook** it seems to **DISABLE** every other _extension_ besides **GoGuardian**.

Their might be another method to **DISABLE** **GoGuardian** instead of the other _extensions_ but I simply listed this as an **Idea**.

**Comment by rwg100** [comment, marked as answer]
if it removes all of the extensions besides the blocking one i dont think it would be very useful unless there might be other extensions but it could be useful for extending the battery life of a chromebook because of other bloat

**Reply by NightScripts-API** [reply]
On version 62 their was a method that allowed Javascript to configure locked google forms and a method like that could cause the extension to remove only GoGuardian.

**Reply by rwg100** [reply]
on most chromebooks javascript is disabled though

**Reply by Aka-but-nice** [reply]
Not moine

**Reply by Aka-but-nice** [reply]
Also ingot is here

**Comment by Aka-but-nice** [comment]
https://ingot106.glitch.me/ingot.js

**Reply by that1kidd0** [reply]
Bro how do you use it and is there one for lightspeed ?

**Reply by CheezeKake77** [reply]
What's this

**Comment by Aka-but-nice** [comment]
> So recently I was trying the method by **Bypassi** which looks something like this:
> 
> **javascript:let shim = false;var ids = prompt("haldlgldplgnggkjaafhelgiaglafanh -- GOGUARDIAN ID").split(",");setInterval(()=>{ids.forEach((id)=> opener.chrome.developerPrivate.updateExtensionConfiguration({extensionId: id, fileAccess: shim}));shim = !shim;}, 1500);**
> 
> For this method it allows the _Extension_ to Turn **OFF** and **ON** the **GoGuardian** _extension_.
> 
> I found this type of glitch it seems where when you enter a **Locked** _Google_ _Forms_ and select the _Window_ _key_ on the **chromebook** it seems to **DISABLE** every other _extension_ besides **GoGuardian**.
> 
> Their might be another method to **DISABLE** **GoGuardian** instead of the other _extensions_ but I simply listed this as an **Idea**.

Can you give me the locked form, so I can experiment

**Comment by Alex2Mayhem** [comment]
I'd like to make a statement on quote "Possible v108 Exploit" It's either it does work or it doesn't.

**Reply by Aka-but-nice** [reply]
Yea, it might, who know?
